From 337ae7414751ce995e06321e23bd322c955d0a67 Mon Sep 17 00:00:00 2001
From: Dave Young <dyoung@redhat.com>
Date: Thu, 19 Apr 2012 12:01:43 +0800
Subject: [PATCH] move cleanup scripts to pre-pivot-cleanup hook

below cleanup scripts is moved:
40network: kill-dhclient.sh
90crypt: crypt-cleanup.sh
90multipath: multipathd-stop.sh
95iscsi: cleanup-iscsi.sh
95nfs: nfsroot-cleanup.sh

Signed-off-by: Dave Young <dyoung@redhat.com>
---
 modules.d/40network/module-setup.sh   |    2 +-
 modules.d/90crypt/module-setup.sh     |    2 +-
 modules.d/90multipath/module-setup.sh |    2 +-
 modules.d/95iscsi/module-setup.sh     |    2 +-
 modules.d/95nfs/module-setup.sh       |    2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/modules.d/40network/module-setup.sh b/modules.d/40network/module-setup.sh
index 253a87e..c2ad815 100755
--- a/modules.d/40network/module-setup.sh
+++ b/modules.d/40network/module-setup.sh
@@ -86,7 +86,7 @@ install() {
     inst_hook cmdline 97 "$moddir/parse-bridge.sh"
     inst_hook cmdline 98 "$moddir/parse-ip-opts.sh"
     inst_hook cmdline 99 "$moddir/parse-ifname.sh"
-    inst_hook pre-pivot 10 "$moddir/kill-dhclient.sh"
+    inst_hook pre-pivot-cleanup 10 "$moddir/kill-dhclient.sh"
 
     _arch=$(uname -m)
 
diff --git a/modules.d/90crypt/module-setup.sh b/modules.d/90crypt/module-setup.sh
index 8fdf32b..4a66516 100755
--- a/modules.d/90crypt/module-setup.sh
+++ b/modules.d/90crypt/module-setup.sh
@@ -48,7 +48,7 @@ install() {
     inst "$moddir"/probe-keydev.sh /sbin/probe-keydev
     inst_hook cmdline 10 "$moddir/parse-keydev.sh"
     inst_hook cmdline 30 "$moddir/parse-crypt.sh"
-    inst_hook pre-pivot 30 "$moddir/crypt-cleanup.sh"
+    inst_hook pre-pivot-cleanup 30 "$moddir/crypt-cleanup.sh"
     inst_simple /etc/crypttab
     inst "$moddir/crypt-lib.sh" "/lib/dracut-crypt-lib.sh"
 }
diff --git a/modules.d/90multipath/module-setup.sh b/modules.d/90multipath/module-setup.sh
index 8a193c4..f044f33 100755
--- a/modules.d/90multipath/module-setup.sh
+++ b/modules.d/90multipath/module-setup.sh
@@ -70,7 +70,7 @@ install() {
     inst_libdir_file "multipath/*"
 
     inst_hook pre-trigger 02 "$moddir/multipathd.sh"
-    inst_hook pre-pivot   02 "$moddir/multipathd-stop.sh"
+    inst_hook pre-pivot-cleanup   02 "$moddir/multipathd-stop.sh"
     inst_rules 40-multipath.rules
 }
 
diff --git a/modules.d/95iscsi/module-setup.sh b/modules.d/95iscsi/module-setup.sh
index b6c1c1f..a001a28 100755
--- a/modules.d/95iscsi/module-setup.sh
+++ b/modules.d/95iscsi/module-setup.sh
@@ -63,7 +63,7 @@ install() {
     inst hostname
     inst iscsi-iname
     inst_hook cmdline 90 "$moddir/parse-iscsiroot.sh"
-    inst_hook pre-pivot 90 "$moddir/cleanup-iscsi.sh"
+    inst_hook pre-pivot-cleanup 90 "$moddir/cleanup-iscsi.sh"
     inst "$moddir/iscsiroot.sh" "/sbin/iscsiroot"
     inst "$moddir/mount-lun.sh" "/bin/mount-lun.sh"
 }
diff --git a/modules.d/95nfs/module-setup.sh b/modules.d/95nfs/module-setup.sh
index 455f52f..3587bdd 100755
--- a/modules.d/95nfs/module-setup.sh
+++ b/modules.d/95nfs/module-setup.sh
@@ -58,7 +58,7 @@ install() {
 
     inst_hook cmdline 90 "$moddir/parse-nfsroot.sh"
     inst_hook pre-udev 99 "$moddir/nfs-start-rpc.sh"
-    inst_hook pre-pivot 99 "$moddir/nfsroot-cleanup.sh"
+    inst_hook pre-pivot-cleanup 99 "$moddir/nfsroot-cleanup.sh"
     inst "$moddir/nfsroot.sh" "/sbin/nfsroot"
     inst "$moddir/nfs-lib.sh" "/lib/nfs-lib.sh"
     mkdir -m 0755 -p "$initdir/var/lib/nfs/rpc_pipefs"
